consul/config_templates/consul.yaml

```diff
@@ -466,55 +454,27 @@ spec:
                 - "/bin/sh"
                 - "-ec"
                 - |
-                  curl http://127.0.0.1:8500/v1/status/leader 2>/dev/null | \
-                  grep -E '".+"'
-
----
-# Source: consul/templates/tests/test-runner.yaml
-apiVersion: v1
-kind: Pod
-metadata:
-  name: "consul-consul-test-1klet"
-  labels:
-    app: consul
-    chart: consul-helm
-    heritage: Tiller
-    release: consul
-  annotations:
-    "helm.sh/hook": test-success
-spec:
-  containers:
-    - name: consul-test
-      image: "consul:1.5.0"
-      env:
-        - name: HOST_IP
-          valueFrom:
-            fieldRef:
-              fieldPath: status.hostIP
-      command:
-        - "/bin/sh"
-        - "-ec"
-        - |
-            export VALUE="i66r7ndr4tjqbyfnh60o5lgo"
-            export CONSUL_HTTP_ADDR="${HOST_IP}:8500"
-            consul kv delete _consul_helm_test
-            consul kv put _consul_helm_test $VALUE
-            [ `consul kv get _consul_helm_test` = "$VALUE" ]
-            consul kv delete _consul_helm_test
-  restartPolicy: Never
-
+                  curl http://127.0.0.1:8500/v1/status/leader \
+                  2>/dev/null | grep -E '".+"'
+          resources:
```

Comment:
I saw the resource limit of the server which we already added into the `value.yaml`. Should we use the configuration in the `value.yaml` to here?

Reply:
I'm not quite sure I understand. The Consul helm chart was used to generate this file, based on the values in `consul-values.yaml` and the templates in the chart.

Reply:
I mean should we add the resource limit to `values.yaml` and import the common config like 

```
resources:
{{ toYaml .Values.resources | indent n }}
  
```

Reply:
@Aisuko, you're highlighting that perhaps we can avoid specifying the resource constraints here statically, but instead leverage the centrally defined constraint values from `values.yaml`, right?

Reply:
@Aisuko, if so, your suggestion makes sense.